package.json

```diff
@@ -32,6 +32,7 @@
     "validator": "^11.1.0"
   },
   "devDependencies": {
+    "concurrently": "^5.0.0",
```

Comment:
I'm a little confused about how this wasn't here? It's in my package-lock.json but not package.json